_includes/package_links.html

```diff
@@ -51,11 +50,13 @@
        Wiki
   </a>
   {%endif%}
-  <a class="{{list_group_class}}"
-     target="_blank" 
+  <a class="{{list_group_class}} {% unless package.snapshot.documented %}disabled{% endunless %}"
+     target="_blank"
+     {% if package.snapshot.documented %}
      href="http://http404error.github.io/roseco/graph.html?id=ros.json&focus={{page.package_name}}&height=1&depth=2&tred=standard&metagroup=false&colorby=Health&direction=LR"
-     title="View RosEco package graph"
-     {% unless package.snapshot.documented %}disabled{% endunless %}>
+     {% endif %}
```

Comment:
This should probably be an else not an endif+unless

Reply:
Sure, either way it's fine.

Reply:
Done in https://github.com/ros-infrastructure/rosindex/pull/219/commits/7fcbabefbc58dcdcc0c1d0b91bda954cf892ca94.